.gitignore

```diff
@@ -1,5 +1,4 @@
-bower_components/
-bower_components-1.x/
-bower-1.x.json
 node_modules
```

Comment:
Removed the `/` here to try to get the modulizer to not add this twice, but seems like it's just dumb 😒